mne_bids/mne_bids.py

```diff
@@ -122,7 +122,7 @@ def _channels_tsv(raw, fname, overwrite=False, verbose=True):
         status.append('bad' if ch in raw.info['bads'] else 'good')
         _channel_type = channel_type(raw.info, idx)
         if _channel_type in get_specific:
-            _channel_type = coil_type(raw.info, idx)
+            _channel_type = coil_type(raw.info, idx, channel_type)
```

Comment:
you are passing a function handle here, this should probably be `_channel_type`
```suggestion
            _channel_type = coil_type(raw.info, idx, _channel_type)
```

Reply:
... or do you want to always fall back on `n/a` if no specific coil type is found?

Reply:
Sorry that was a mistake. Good catch!